engine/src/main/java/org/terasology/math/AABB.java

```diff
@@ -452,11 +452,11 @@ public int hashCode() {
     }
 
     public boolean intersectRectangle(Vector3f from, Vector3f direction) {
-        Vector3f dirfrac = new Vector3f();
-
-        dirfrac.x = 1.0f / direction.x;
-        dirfrac.y = 1.0f / direction.y;
-        dirfrac.z = 1.0f / direction.z;
+        Vector3f dirfrac = new Vector3f(
```

Comment:
Let's rename dirfrac to dirFrac, shall we?

(i realize you did not introduce this variable in the first place, but it never hurts to sprinkle some code-quality-dust over areas that you visit)